iron-iconset-svg.html

```diff
@@ -225,8 +230,9 @@
           cssText += '-webkit-transform:scale(-1,1);transform:scale(-1,1);';
         }
 
-        svg.setAttribute('viewBox', viewBox);
-        svg.setAttribute('preserveAspectRatio', 'xMidYMid meet');
+        svg.setAttributeNS(null, 'viewBox', viewBox);
```

Comment:
Why is this change needed? Not questioning, just trying to understand since they look the same to me :)